pom.xml

```diff
@@ -11,6 +11,7 @@
     <modules>
```

Comment:
too much to make alchemy extend dw_parent pom?

Reply:
++

Reply:
@mmartes @JliptonRTR my assumption was that we didnt want to use that for the public project. I confirmed this with Gene thus we need to continue not using dw_parent_pom in public projects.

alchemy-rtr does use dw_parent_pom. I'll work on that one next

Reply:
Ah i forgot alchemy is public